### DISCUSSION TITLE
Facing issues after deployed to vercel

### DISCUSSION THREAD

**Discussion by FSE-HYD-Raju** [blitz-js/blitz#2714] [General]
Hi Team,

I developed a blitz app and deployed it to vercel. I am trying to access the application. When I sign up got the error saying TypeError: Cannot read property '$create' of undefined. But the user got created in db. Also I tried logging in with the created user, Getting the same error. I am not getting what I am missing. Please help.

![image](https://user-images.githubusercontent.com/9348371/133147517-943e7ab6-b465-48cd-a97b-d703f94ec6dd.png)

**Comment by flybayer** [comment]
Issue here: https://github.com/blitz-js/blitz/issues/2716
